packages/plugin-lighthouse/src/lib/normalize-flags.unit.test.ts

```diff
@@ -122,4 +119,16 @@ describe('normalizeFlags', () => {
     ).toEqual(expect.not.objectContaining({ 'list-all-audits': true }));
     expect(getLogMessages(ui().logger)).toHaveLength(1);
   });
+
+  it('should remove any flag with an empty array as a value', () => {
+    const flags = {
+      onlyAudits: [],
+      skipAudits: [],
+      onlyCategories: [],
+    };
+    const result = normalizeFlags(flags);
+    expect(result).not.toHaveProperty(['onlyAudits']);
```

Comment:
Minor mistake - but otherwise nice test :+1: 

```suggestion
    expect(result).not.toHaveProperty('onlyAudits');
```

Reply:
Thanks for pointing it out! Don't know how I missed that